internal/pkg/service/common/dependencies/public.go

```diff
@@ -76,23 +76,23 @@ func newPublicScope(ctx context.Context, baseScp BaseScope, storageAPIHost strin
 	var indexWithComponents *keboola.IndexComponents
 	if cfg.preloadComponents {
 		logger.Info(ctx, "loading Storage API index with components")
-		indexWithComponents, err = keboola.APIIndexWithComponents(ctx, storageAPIHost, keboola.WithClient(&baseHTTPClient))
+		indexWithComponents, err = keboola.APIIndexWithComponents(ctx, storageAPIHost, keboola.WithClient(&baseHTTPClient), keboola.WithOnSuccessTimeout(1*time.Minute))
```

Comment:
How long should be timeout for API related `dependencies`?

Reply:
The response may have up to `1-2` MBs, 30s?